### ISSUE TITLE
Add Multi-App Run for Kubernetes to the Distributed Calculator Tutorial

### ISSUE THREAD

**Issue by msfussell** [dapr/quickstarts#916]
Add Multi-App Run for Kubernetes to the Distributed Calculator Tutorial
This one https://github.com/dapr/quickstarts/tree/master/tutorials/distributed-calculator

**Comment by mukundansundar** [comment]
example PR for template that can be used for multi-app run k8s https://github.com/dapr/quickstarts/pull/926

**Comment by sicoyle** [comment]
@MeeraTresa Hope you are enjoying the Grace Hopper Conference Open Source Day event! Happy to have you here with us. Please let us know if you need any help along the way with this issue😁

**Comment by MeeraTresa** [comment]
Yes, Sam !! Thank you for having me. I think I'll take some time setting this up. I'll  definitely reach out here/slack/discord as I work through this.
